src/system/AdminModule/Resources/views/AdminInterface/breadcrumbs.html.twig

```diff
@@ -0,0 +1,9 @@
+<ol class="breadcrumb">
+    <li>{{ __('You are in:') }} <a href="{{ path('zikulaadminmodule_admin_adminpanel') }}">{{ __('Administration') }}</a></li>    
+    <li><a href="{{ path('zikulaadminmodule_admin_adminpanel', {'acid': caller.category.cid}) }}">{{ caller.category.name }}</a></li>
+    <li><a href="#" class="text-capitalize">{{ caller.info.displayname }}</a></li>
+        {% if caller._zkType != 'admin' %}
+        <li><a href="#" class="text-capitalize">{{ caller._zkType }}</a></li>
```

Comment:
I disagree - the whole `if/endif` block could be *out*dented so the `<li>` line up the same. no?